scripts/assign.py

```
@@ -207,34 +207,110 @@ def main():
+
+                    # If no PRs found via GraphQL, try REST API fallback
+                    if not has_pr:
+                        try:
+                            # Check title and body references for linked PRs
+                            search_url = "https://api.github.com/search/issues"
+                            search_query = (
+                                f"type:pr state:open repo:{owner}/{repo} {current_issue_number} in:title,body"
+                            )
+                            search_params = {"q": search_query}
+                            print(f"Searching PRs with REST API query: {search_query}")
+                            search_response = requests.get(search_url, headers=headers, params=search_params)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)